sql/hive/src/main/scala/org/apache/spark/sql/hive/client/IsolatedClientLoader.scala

```diff
@@ -174,8 +174,6 @@ private[hive] class IsolatedClientLoader(
 
   // Pre-reflective instantiation setup.
   logDebug("Initializing the logger to avoid disaster...")
-  Thread.currentThread.setContextClassLoader(classLoader)
-
```

Comment:
It is redundant here since the `contexClasstloader` would be reset correctly within the constructor of `ClientWrapper`, and setting the contextClassLoader too early here would prevent us to retrieve the other classLoader within the same thread which having `SessionState` containing the user input option.